extensions/amp-mraid/0.1/amp-mraid.js

```diff
@@ -66,11 +66,6 @@ export class MraidInitializer {
     /** @private {boolean} */
     this.fallback_ = false;
 
-    if (getMode().runtime !== 'inabox') {
-      dev().error(TAG, 'Only supported with Inabox');
-      return;
-    }
-
     const ampMraidScripts = this.ampdoc_.getHeadNode().querySelectorAll(
```

Comment:
Also @jeffkaufman I'm not sure if we will be able to get mraid script within AMP page. And why do we still need the check given that we check it in validator.

Reply:
Yes, the validator will make sure we don't see amp-mraid on AMP pages, but we might see it in AMP ads served to AMP pages.  This PR makes us start respecting fallback configuration in that case.

Reply:
ah, another trouble after making amp-mraid valid AMP. It blocks the visibility measurement while waiting for the unneeded amp-mraid script. In general, this should be considered an error case (serving amp-mraid ad on AMP page). 

I agree with you that your fallback change is the best we can do here.